api/managementpb/postgresql.proto

```diff
@@ -78,6 +78,12 @@ message AddPostgreSQLRequest {
   MetricsMode metrics_mode = 21;
   // List of collector names to disable in this exporter.
   repeated string disable_collectors = 22;
+  // TLS CA certificate.
+  string tls_ca = 23;
+  // TLS Certifcate.
+  string tls_cert = 24;
+  // TLS Certificate Key.
+  string tls_key = 25;
```

Comment:
ok, here we have a problem 😄 we were saying that we use same naming that's used on upstream, but looks like it was false for PostgreSQL, because upstream uses SSL everywhere, but we use TLS wording. Any ideas how can we fix this problem, without loosing backward compatibility?